jwql/shared_tasks/shared_tasks.py

```diff
@@ -528,7 +569,7 @@ def prep_file(input_file, in_ext):
     input_path, input_name = os.path.split(input_file)
     logging.info("\tPath is {}, file is {}".format(input_path, input_name))
 
-    if "uncal" not in in_ext:
```

Comment:
I feel as if we can probably remove this completely, now that the pipeline has been modified to (as far as I can see) be callable starting with any input file.

Reply:
I didn't test with any input file other than dark.fits. Actually though, the only possible input files, since they all ultimately come from MAST, are uncal and dark. Beyond those, any other input files (rate, cal, etc) would never need to have the calwebb_detector1 pipeline called at all.

Reply:
Thinking a little more deeply about this, I agree. I think it's safe to remove this block completely.

Reply:
Removed.